modules/kinesis/src/it/scala/com/snowplowanalytics/snowplow/enrich/kinesis/utils.scala

```diff
@@ -30,6 +30,7 @@ import com.snowplowanalytics.snowplow.enrich.common.fs2.config.io.Input
 import com.snowplowanalytics.snowplow.enrich.common.fs2.test.Utils
 
 object utils extends CatsIO {
+  val maxRecordSize = 1024 * 1024
```

Comment:
This could be received as an argument for `mkEnrichPipe`, then, we'll be able to set a low size when testing the record limit